Style Guide and Diataxis Review: Hornet II (#1322)
gohornet/hornet: documentation/docs/getting_started/nodes_101.md

Like any distributed system, nodes in an IOTA network synchronize their databases with other nodes called neighbors to form a
single source of truth.

When one node, no matter where it is in the world, receives a message, it will try to _gossip_ it to all its neighbors. This way, all of the nodes will eventually see all the messages, and store them in their local databases.

> To synchronize, nodes in IOTA networks use milestones. If the node has the history of messages that a milestone references, that milestone is solid. Therefore, nodes know if they are synchronized if the index of their latest solid milestone is the same as the index of the latest milestone that it has received.

Reviewer: ```suggestion
Nodes in IOTA networks use milestones to synchronize. If a node has the history of messages that a milestone references, that milestone is solid. Therefore, a node knows if it is synchronized if its latest solid milestone index is the same as the index of the latest milestone that it has received.
```
